tests/authentication/05-full-control.t

```diff
@@ -20,6 +20,8 @@
 . "$(dirname "$0")/test_header"
 set_test_number 12
 
+API_VERSION='5'
```

Comment:
Perhaps:

```bash
API_VERSION="$(python -c 'from cylc.flow.network import API; print(API)')"
```

Reply:
Fixed.. thought about it, but didn't follow through for w/e reason.

Reply:
Fixed.. thought about it, but didn't follow through for w/e reason.